neworigins/extra.cpp

```diff
@@ -728,6 +728,27 @@ Faction *Game::CheckVictory()
 		}
 	}
 
+	// Freezing effect
+	forlist_reuse(&regions) {
+		ARegion *r = (ARegion *) elem;
+
+		// No effect for regions with clear skies
+		if (r->clearskies) {
+			continue;
+		}
+
+		// Check if regions in freezing zome in surface
+		if (r->zloc == ARegionArray::LEVEL_SURFACE && (r->yloc <= 0 || r->yloc >= 71)) {
+			r->Pillage();
+			r->SetWeather(W_BLIZZARD);
+			printf("Freeze (%d,%d) region in %s of %s\n",
+				r->xloc, r->yloc, r->name->Str(), TerrainDefs[TerrainDefs[r->type].similar_type].name
+			);
+		}
+
+		// TODO: Check if regions in freezing zome in UW
```

Comment:
I thought you’d previously said the UW wouldn’t freeze?

Reply:
I do not think I said this :)

Reply:
Okay.. this (from the #win-conditions channel) is what led me to think it wouldn't *really* affect the underworld.

```Underworld cities do not contribute their vote (as they are safe from The Calamity (or they think so)```

I agree that it does imply that it might, but seemed to also imply it might not and I think I was just hopeful :)